crates/task/src/task_template.rs

```diff
@@ -143,13 +143,13 @@ impl TaskTemplate {
             &variable_names,
             &mut substituted_variables,
         )?;
-        let command = substitute_all_template_variables_in_str(
+        let _command = substitute_all_template_variables_in_str(
```

Comment:
Can we just remove those?

Reply:
Ohh, missed the substitution side effect, sorry.
But still — I think we'd want to reuse the substituted label at least, somewhere.